repositories/riotclient.go

```diff
@@ -0,0 +1,46 @@
+package repositories
+
+import (
+	"context"
+	"net/http"
+	"strings"
+)
+
+type rgapiClient struct {
```

Comment:
Should probably rename this file

Reply:
Renamed as `client.go` (the namespace of folder is now `rgapis\client.go`)